dlme_airflow/utils/add_thumbnails.py

```diff
@@ -0,0 +1,41 @@
+# /bin/python
+import os
+import pandas
+import logging
+
+from typing import Optional
+from dlme_airflow.utils.schema import get_schema
+from dlme_airflow.utils.catalog import catalog_for_provider
+
+
+def add_thumbnails(**kwargs) -> None:
+    "Add a thumbnail column based on the value in the url column"
+    provider = catalog_for_provider(kwargs["provider"])
+    coll = provider[kwargs["collection"]]
```

Comment:
The collection model should be available in the kwargs.

Reply:
It's actually a string, because of `post_harvest.build_post_havest_task()` (sic) works:

https://github.com/sul-dlss/dlme-airflow/blob/main/dlme_airflow/tasks/post_harvest.py#L29

Do you want me to update that?

Reply:
The post-harvest tasks have been updated to use the collection object.